build.gradle

```diff
@@ -0,0 +1,57 @@
+plugins {
```

Comment:
Would you like to try installing the checkstyle plugin for Gradle? Here's the [link to the guide](https://se-education.org/guides/tutorials/checkstyle.html).